tests/acl_loader_test.py

```diff
@@ -150,7 +150,6 @@ def test_icmp_translation(self, acl_loader):
     def test_icmpv6_translation(self, acl_loader):
         acl_loader.rules_info = {}
         acl_loader.load_rules_from_file(os.path.join(test_path, 'acl_input/acl1.json'))
-        print(acl_loader.rules_info)
```

Comment:
why remove this line?

Reply:
It's a debug line and actually `pytest` doesn't show the output of `print` statement.